### PR TITLE
AML - Idle Shutdown test 

### PR DESCRIPTION
**Pull request by batemansogq**
## PR Summary

Additional AML test covering idle shutdown control on compute instances. 
The first update from the secure AML design work. 

## PR Checklist

- [ ] PR has a meaningful title
- [ ] Summarized changes
- [ ] Change is not breaking
- [ ] This PR is ready to merge and is not **Work in Progress**
- **Rule changes**
  - [ ] Unit tests created/ updated
  - [ ] Rule documentation created/ updated
  - [ ] Link to a filed issue
  - [ ] [Change log](https://github.com/Azure/PSRule.Rules.Azure/blob/main/docs/CHANGELOG-v1.md) has been updated with change under unreleased section
- **Other code changes**
  - [ ] Unit tests created/ updated
  - [ ] Link to a filed issue
  - [ ] [Change log](https://github.com/Azure/PSRule.Rules.Azure/blob/main/docs/CHANGELOG-v1.md) has been updated with change under unreleased section

### CODE REVIEW

**Review by BernieWhite — CHANGES_REQUESTED** [review]
Awesome work @batemansogq. I made a few minor tweaks and have a few suggestions below for you to review. Any questions let me know.

File: docs/en/rules/Azure.ML.ComputeIdleShutdown.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+category: Provision
+resource: ML
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.ML.ComputeIdleShutdown/
+---
+
+# ML Compute Idle Shutdown
+
+## SYNOPSIS
+
+Use ML - Compute Instances configured for idle shutdown.
```
[the remark this answers is not in the document]
Comment (batemansogq) [reply docs/en/rules/Azure.ML.ComputeIdleShutdown.md:14]: good.

**Comment by nfmoore** [comment]
The rules look good @batemansogq. Don't see any issues.

**Comment by BernieWhite** [comment]
Replaced by #2500

**Pull request closed without merging**